distributed/stealing.py

```diff
@@ -74,17 +74,16 @@ def transition(
         ts = self.scheduler.tasks[key]
         if finish == "processing":
             self.put_key_in_stealable(ts)
-
-        if start == "processing":
+        elif start == "processing":
```

Comment:
IIUC these can't both be true. So changed this to `elif`

Reply:
Indeed 
https://github.com/dask/distributed/blob/e98d57dcad72d4a87fb177c4b2b9c98b2b2c50e1/distributed/scheduler.py#L5518-L5519

out of curiosity, is `elif` faster?

Reply:
If one matches the first case initially, yes. Though admittedly it is not a huge difference

```python
In [1]: %%timeit s = "a"
   ...: if s == "a":
   ...:     pass
   ...: elif s == "b":
   ...:     pass
   ...: 
   ...: 
20.9 ns ± 0.166 ns per loop (mean ± std. dev. of 7 runs, 10000000 loops each)

In [2]: %%timeit s = "a"
   ...: if s == "a":
   ...:     pass
   ...: if s == "b":
   ...:     pass
   ...: 
   ...: 
39.3 ns ± 0.172 ns per loop (mean ± std. dev. of 7 runs, 10000000 loops each)
```